pages.de/common/steamcmd.md

```diff
@@ -0,0 +1,16 @@
+# steamcmd
+
+> Ein Kommandozeilenwerkzeug, um über Steam verfügbare Anwendungen zu verwalten.
+> Weitere Informationen: <https://manned.org/steamcmd>.
+
+- Installiere und aktualisiere eine Anwendung ohne dich einzuloggen:
+
+`steamcmd +login anonymous +app_update {{anwendungs_id}} +quit`
+
+- Installiere oder aktualisiere eine Anwendung unter Angabe deiner Zugangsdaten:
+
+`steamcmd +login {{benutzername}} +app_update {{anwendungs_id}} +quit`
+
+- Installiere eine Anwendung für eine bestimmte Plattform:
+
+`steamcmd +@sSteamCmdForcePlatformType {{windows}} +login {{benutzername}} +app_update {{anwendungs_id}} validate +quit`
```

Comment:
Update command argument for login to `anonymous`'s German equivalent.

Reply:
Here I would argue, that either we use `anonymous` as a constant (without the braces), or we use the concept of a login with an username established with the command before. Once we resolved the first discussion, we can apply the results to this command too 👍